README.md

```diff
@@ -995,6 +997,124 @@ Uninstalling is easy, too:
 $ rake templates:uninstall
 ```
 
+## Configuring Quick
+The Quick configuration object exposes custom options to alter the behavior of
+the framework. The intention of this hook is to provide a global configuration
+to meet the needs of your project. *QuickConfiguration* is never meant to be
+instantiated. If you do so, an exception will be raised. Instead, subclass it
+and override the `configure()` class function, like so:
+
+```swift
+// Swift
+
+import Quick
+import Nimble
+
+class ProjectDataTestConfiguration: QuickConfiguration {
+  override class func configure(configuration : Configuration) {
+    // set options on the configuration object
+  }
+}
+```
+
+```objc
+// Objective-C
+
+#import <Quick/Quick.h>
+#import <Nimble/Nimble.h>
+
+QuickConfigurationBegin(ProjectDataTestConfiguration)
+
++ (void)configure:(Configuration *configuration) {
+  // set options of the configuration object
+}
+
+QuickConfigurationEnd
+```
+
+Projects may include several configurations. However, Quick does not make any
+guarantee about the order in which they are executed.
+
+### Adding Global Before and After Filters
+Similar to `beforeEach()` and `afterEach()` functions in QuickSpec(s), global
+filters can be applied to all tests. For example, a computed value may need to
+be reset before each test case. In the following case, it is assumed that the
+tests rely on the `height` property of the `Dorsal` singleton to be 0 in order
+to do some sort of mathematical computation.
+
+```swift
+// Swift
+
+import Quick
+import Nimble
+
+class FinConfiguration: QuickConfiguration {
+  override class func configure(configuration: Configuration) {
+    configuration.beforeEach() {
+      fin = Dorsal.sharedFin()
+      fin.height = 0
+    }
+  }
+}
+```
+
+```objc
+// Objective-C
+
+#import <Quick/Quick.h>
+#import "Dorsal.h"
+
+QuickConfigurationBegin(FinConfiguration)
+
++ (void) configure:(Configuration *)configuration {
```

Comment:
nitpick: Remove space between `(void)` and `configure:`. The other examples have no space, and I prefer that.